### PR TITLE
Attempt to address comments from Warren Kumari

### PR DESCRIPTION
**Pull request by gorryfair**
Proposed resolutions

### CODE REVIEW

File: draft-ietf-ccwg-rfc5033bis.md [hunk 1]
```diff
@@ -249,17 +249,24 @@ degree of certainty of those answers.
 
 Congestion control algorithms without empirical evidence of Internet-scale
 deployment SHOULD seek Experimental status.
+Specifications published as Experimental ought to explain the reason for
+the status and what further information would be required to progress to
+standards track. For example, section 12. of  RFC6928 provides
```
Comment (martinduke) [inline draft-ietf-ccwg-rfc5033bis.md:254]:
```suggestion
standards track. For example, section 12 of RFC6928 provides
```

Comment (martinduke) [reply draft-ietf-ccwg-rfc5033bis.md:254]: Should this be an informational reference?

File: draft-ietf-ccwg-rfc5033bis.md [hunk 2]
```diff
@@ -249,17 +249,24 @@ degree of certainty of those answers.
...
+“Usage and Deployment Recommendations” that describe the experiments
+expected by the TCPM working group. Section 4 of RFC4614 provides other
```
Comment (martinduke) [inline draft-ietf-ccwg-rfc5033bis.md:256]: 4614 has been obsoleted by 7414, but the later document is not as careful about formal Experimental status as the earlier one, so maybe your choice to reference 4614 is right.

But maybe this should be a reference?

File: draft-ietf-ccwg-rfc5033bis.md [hunk 3]
```diff
@@ -325,8 +332,12 @@ criteria in the remainder of this document might not apply.
 
 # Evaluation Criteria {#evaluation-criteria}
 
-As noted above, authors are expected to do a well-rounded evaluation of the pros
-and cons of congestion control algorithms that are brought to the IETF. The
+Authors are expected to evaluate the congestion control algorithms that
+they bring to the IETF.
```
Comment (martinduke) [inline draft-ietf-ccwg-rfc5033bis.md:336]: IMO this paragraph pins too much on the authors. "Authors are expected to evaluate..." and then the second sentence uses the passive voice. I would prefer a paragraph like so:

"Publication of a specification by the IETF requires a well-rounded evaluation of the pros and cons of an algorithm. Both the document authors and participants in an IETF Working Group can conduct parts of this evaluation."

That's all we need to say, but I wouldn't object to further advice saying that the more work the authors do, the more likely it is to progress.

**Review by martinduke — CHANGES_REQUESTED** [review]

**Review by martinduke — APPROVED** [review]

File: draft-ietf-ccwg-rfc5033bis.md [hunk 4]
```diff
@@ -325,8 +332,12 @@ criteria in the remainder of this document might not apply.
...
+A well-rounded evaluation of the pros and cons of an algorithm is expected
+by a working group before it submits the specification for publication by the IETF.
+
```
Comment (huitema) [inline draft-ietf-ccwg-rfc5033bis.md:339]: You made sure to use the active form in the previous sentence, and now we are back to passive. How about:

A working group is expected to perform a well-rounded evaluation of the pros and cons of an algorithm
before it submits the specification for publication by the IETF.

**Review by huitema — APPROVED** [review]
You could merge it as is, but I don't like the remaining passive form too much.

**Pull request merged**